haminfo/mqtt/processors.py

```
@@ -0,0 +1,309 @@
+            tic = time.perf_counter()
+            self.session.bulk_save_objects(self.aprs_packets)
+            self.session.commit()
+            toc = time.perf_counter()
+
+            with self.stats_lock:
+                self.stats['packets_saved'] = (
+                    self.stats.get('packets_saved', 0) + packets_to_save
+                )
+
+            logger.info(f'Time to save APRS packets = {toc - tic:0.4f}s')
+        except Exception as ex:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)